doc/design/vic-machine.md

```diff
@@ -73,7 +73,8 @@ These are various examples intended to illustrate how vic-machine is intended to
 vic-machine create
 -TO=https://root@****:vcenter/
```

Comment:
Just find that this target URL is different with what we're using with govc. Is that by design?
I'm just wondering if we're introducing too many resource format to user. Cause user will need govc to figure out the resource pool format.